package.json

```diff
@@ -11,6 +11,8 @@
     "examples": "npm run build && http-server",
     "test": "nyc mocha -r ts-node/register -r source-map-support/register -r jsdom-global/register test/*.spec.ts test/**/*.spec.ts",
     "test:fullstory": "nyc mocha -r ts-node/register -r source-map-support/register -r jsdom-global/register test/fullstory.spec.ts",
+    "test:browser": "PLAYWRIGHT_BROWSERS_PATH=0 nyc mocha -r ts-node/register -r source-map-support/register -r jsdom-global/register test/*.spec.ts -- --grep 'CEDDL to FullStory'",
```

Comment:
Couple notes here:
- `PLAYWRIGHT_BROWSERS_PATH=0` installs browser binaries in `node_modules/playwright-core/.local-browsers`. I took this route to isolate the impact of DLO Playwright browser tests on the broader dev environment. Interested in feedback on this approach.
- I'm expecting `--grep 'CEDDL to FullStory'` to evolve in subsequent PRs to hybridize other ruleset tests for execution in node or browser environments.